pkgs/development/python-modules/pygobject/3.nix

```diff
@@ -19,7 +19,8 @@ buildPythonPackage rec {
   nativeBuildInputs = [ pkgconfig meson ninja gobject-introspection ];
   buildInputs = [ glib gobject-introspection ]
                  ++ stdenv.lib.optionals stdenv.isDarwin [ which ncurses ];
-  requiredPythonModules = [ pycairo cairo ];
+  propagatedBuildInputs = [ cairo ];
```

Comment:
@jtojnar should we propagate this dep you think? I prefer to avoid it, but did it now just to be safe

Reply:
I do not see why it should be propagated. Gdk overrides mention Cairo typelib but that is only for old versions of Gtk and the typelib is in `gobject-introspection` package anyway. The only reason it would be propagated is for compilation but it is not listed in pkg-config file so I doubt it is the case.

It was propagated in https://github.com/NixOS/nixpkgs/pull/5893 without any explanation.